src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -340,6 +425,17 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
         }
     }
 
+    updateGitopsUrl(value: string): void {
+        this.setState({
+            form: {
+                ...this.state.form,
+                host: value,
```

Comment:
Try to use prevState syntax

Reply:
will take care next time

Reply:
Please address this in this PR only. Minor change.